testresultparser/junit_reports.go

```diff
@@ -3,115 +3,134 @@ package testresultparser
 import "encoding/xml"
 
 type report struct {
-    TestSuite   string     `xml:"name,attr"`
-    Tests       int        `xml:"tests,attr"`
-    Failures    int        `xml:"failures,attr"`
-    Errors      int        `xml:"errors,attr"`
-    Skipped     int        `xml:"skipped,attr"`
-    Time        string     `xml:"time,attr"`
-    TestResults []testCase `xml:"testcase"`
+	TestSuite   string     `xml:"name,attr"`
+	Tests       int        `xml:"tests,attr"`
+	Failures    int        `xml:"failures,attr"`
+	Errors      int        `xml:"errors,attr"`
+	Skipped     int        `xml:"skipped,attr"`
+	Time        string     `xml:"time,attr"`
+	TestResults []testCase `xml:"testcase"`
+	StdOut      systemOut  `xml:"system-out"`
```

Comment:
`system-out` and `system-err` are separate elements in the XML, so it needs a separate struct. 
Combining it doesn't work. Alternatively, I could use a nested struct like,
```type report struct {
	TestSuite   string     `xml:"name,attr"`
	Tests       float64    `xml:"tests,attr"`
	Failures    float64    `xml:"failures,attr"`
	Errors      float64    `xml:"errors,attr"`
	Skipped     float64    `xml:"skipped,attr"`
	Time        string     `xml:"time,attr"`
	TestResults []testCase `xml:"testcase"`
	StdOut      struct {
		SystemOut string `xml:",chardata"`
	} `xml:"system-out"`
	StdErr struct {
		SystemErr string `xml:",chardata"`
	} `xml:"system-err"`
}
```
But, I would prefer it outside because it is reused that way.